src/flb_metrics_exporter.c

```diff
@@ -299,6 +299,15 @@ struct cmt *flb_me_get_cmetrics(struct flb_config *ctx)
         }
     }
 
+    if (ctx->log != NULL) {
+        ret = cmt_cat(cmt, ctx->log->metrics->cmt);
+        if (ret == -1) {
+            flb_error("[metrics exporter] could not append global log_metrics_ctx");
```

Comment:
Please change this so the message reflect what it is rather than the context name.

Reply:
Done